lib/ex_plasma/output.ex

```diff
@@ -170,20 +174,20 @@ defmodule ExPlasma.Output do
 
   # Validate the output ID. Bypass the validation if it doesn't
   # exist in the output body.
-  defp do_validate_id(%{output_id: nil} = output), do: {:ok, output}
+  defp do_validate_id(%__MODULE__{output_id: nil} = output), do: {:ok, output}
 
-  defp do_validate_id(%{output_id: %{} = output_id}),
-    do: ExPlasma.Output.Position.validate(output_id)
+  defp do_validate_id(%__MODULE__{output_id: %{} = output_id}),
```

Comment:
```suggestion
  defp do_validate_id(%__MODULE__{output_id: %{} = output_id}), do: Position.validate(output_id)
```

if you want to do this, maybe it's worth adjusting the formatter and credo by adding 
`  line_length: 120,`

` {Credo.Check.Readability.MaxLineLength, [priority: :low, max_length: 120]},`
same as in childchain!

Reply:
maybe it already works and you need to convince the formatter

Reply:
Already have it: https://github.com/omisego/ex_plasma/blob/master/.credo.exs#L99